docs/user/introduction.asciidoc

```diff
@@ -1,12 +1,161 @@
-code, no additional infrastructure required.
+* **An open-source analytics and visualization platform.**
+Use {kib} to explore your {es} data, and then build beautiful visualizations and dashboards.
+
+* **A UI for managing the Elastic Stack.**
+Manage your security settings, assign user roles, take snapshots, roll up your data,
+and more &mdash; all from the convenience of a {kib} UI.
+
+* **A centralized hub for Elastic's solutions.** From log analytics to
+document discovery to SIEM, {kib} is the portal for accessing these and other capabilities.
+
+[role="screenshot"]
+image::images/intro-kibana.png[]
+
+[float]
+[[get-data-into-kibana]]
+=== Getting data into {kib}
+
+{kib} is designed to use {es} as a data source. Think of Elasticsearch as the engine that stores
+and processes the data, with {kib} sitting on top.
+
+From the home page, {kib} provides these options for getting data in:
+
+* <<add-sample-data, Add a sample data set>> and take {kib} for a test drive without loading data yourself.
+* Set up a data flow to Elasticsearch using our built-in tutorials.
+(If a tutorial doesn’t exist for your data, go to the
+{beats-ref}/beats-reference.html[Beats overview] to learn about other data shippers
+in the {beats} family.)
+* Import static data using the file upload feature.
+* {ref}/getting-started-index.html[Index your data] into Elasticsearch with REST APIs or client libraries.
++
+[role="screenshot"]
+image::images/intro-data-tutorial.png[Ways to get data in from the home page]
+
+
+{kib} uses an
+<<index-patterns, index pattern>> to tell it which {es} indices to explore.
+If you add sample data or run a built-in tutorial, you get an index pattern for free,
+and are good to start exploring. If you load your own data, you can create
+an index pattern in <<management, Management>>.
+
+[float]
+[[explore-and-query]]
+=== Explore & query
+
+Ready to dive into your data? With <<discover, Discover>>, you can explore your data and
+search for hidden insights and relationships. Ask your questions, and then
+narrow the results to just the data you want.
+
+[role="screenshot"]
+image::images/intro-discover.png[]
+
+[float]
+[[visualize-and-analyze]]
+=== Visualize & analyze
+
+A visualization is worth a thousand log lines, and {kib} provides
+many options for showcasing your data. Use <<lens, Lens>> to analyze your data
+with classic charts, tables, metrics, and more. If there
+is a better visualization for your data, Lens suggests it, allowing for quick
+switching between visualization types.
+
+Once your visualizations are just the way you want,
+use <<dashboard, Dashboard>> to collect them in one place. A dashboard provides
+insights into your data from multiple perspectives, and you can watch the data
+change in real time.
+
+[role="screenshot"]
+image::images/intro-dashboard.png[]
+
+{kib} also offers these features that address specific visualization needs.
+
+* <<visualize, Visualize>> allows you to display your data in
```

Comment:
Right now we make mention of Lens in the intro of Visualize & Analyze, but we don't mention it again any where else in the rest of the larger intro.  I know it breaks up the 2 sentence pattern of the bullets, but maybe for the visualize bullet we add a middle sentence that just says "It's also home to Lens, a drag-and-drop interface for rapid ad-hoc chart building." If we use that verbiage we'll need to start the third sentence differently by taking out the "also."